projects/tags-input-demo/src/app/users-list-item/users-list-item.component.scss

```diff
@@ -0,0 +1,27 @@
+.item {
+    cursor: pointer;
+    padding: 0.50rem 1.5rem;
+    transition: all ease 0.5s;
+    display: flex;
+    align-items: center;
+    &:hover {
+      background: #d0b0b0;
```

Comment:
I'd suggest moving these hard-coded colors to a variable.scss file and importing it here.
So these colors can be re-used.

Reply:
fixed